.env.sample

```diff
@@ -119,4 +119,5 @@ FF_TFM_DEAL_CANCELLATION_ENABLED=false
 FF_PORTAL_FACILITY_AMENDMENTS_ENABLED=false
 FF_TFM_SSO_ENABLED=false
 FF_FEE_RECORD_CORRECTION_ENABLED=false
+AUTOMATIC_SALESFORCE_CUSTOMER_CREATION_ENABLED=false
```

Comment:
I should have picked this before, can this be changed to `FF_SALESFORCE_CUSTOMER_CREATION` please. Apologies.